interceptors/1.2/_index.md

```diff
@@ -33,4 +33,4 @@ and other managed classes.
 
 # Compatible Implementations
 
-* [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
\ No newline at end of file
+* [Eclipse GlassFish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads)
```

Comment:
I would prefer this download page:
https://eclipse-ee4j.github.io/glassfish/download
It makes it easier to navigate to all the other GlassFish documentation.